src/scratchpad.js

```diff
@@ -132,7 +132,26 @@ export default function Scratchpad (DOM, props) {
     }
   };
 
+  const clientWidth$ = DOM.observable.pluck('clientWidth');
+
+  const mouseDowns$ = DOM.select('.handler').events('mousedown');
+  const mouseUps$ = DOM.select('.tricycle').events('mouseup');
+  const mouseMoves$ = DOM.select('.tricycle').events('mousemove');
+  const mouseLeaves$ = DOM.select('.tricycle').events('mouseleave');
+
+  const windowSizes$ = mouseDowns$
+    .flatMap(mouseDown => mouseMoves$.takeUntil(mouseUps$.merge(mouseLeaves$)))
+    .combineLatest(clientWidth$.throttle(100))
```

Comment:
Whenever you have `a$.combineLatest(b$, ...).map(([a, b]) => ...)` you can collapse it to `a$.combineLatest(b$, (a, b) => ...)`.

Each operator introduces a bit of overhead so using the selector function on combineLatest is quite a bit faster.